src/main/resources/assets/tfc/advancements/world/kill_bear.json

```diff
@@ -0,0 +1,27 @@
+{
+    "display": {
+        "icon": {
+            "item": "minecraft:coal",
+            "data": 1
+        },
+        "title": {
+            "translate": "advancements.tfc.world.kill_bear.title"
+        },
+        "description": {
+            "translate": "advancements.tfc.world.kill_bear.description"
+        }
+    },
+    "parent": "tfc:world/root",
+    "criteria":{
+        "native_copper": {
+            "trigger": "minecraft:player_killed_entity",
+            "conditions": {
+                "entity": [
```

Comment:
While testing, this file crashed. Looking further, this is because here we have an extra `[` and `]` brackets. just remove them:
```java
"entity": {
    "type": "tfc:beartfc"
}
```

Reply:
Resolved. Along the ways I noticed the icon was wrong, so fixed that too.